internal/fs/local_modifications_test.go

```diff
@@ -2040,6 +2039,9 @@ func (t *FileTest) Sync_NotDirty() {
 	// Create a file.
 	t.f1, err = os.Create(path.Join(mntDir, "foo"))
 	AssertEq(nil, err)
+	// Sync the file.
```

Comment:
nit: New line before.

Reply:
sync and create are one operation here to create file in GCS. Fixed the new lines in further steps to group operations.